Added a few websites from a deprecated Blocklist

jenngra505 opened the pull request:
There was an AI Blocklist on Codeberg that seems to have disappeared. I had an archive of it, so I added the sites that were on it that weren't here to the list. I also attempted to amp up security for known sites like alkaidvision and kakaobrain on the UBlacklist lists.
Sites added:  ebsynth.com, ishencai.com, metademolab.com, softgist.com, vondy.com, aitoolhunt.com, mascots.ai, gab.ai, idesigns.shop, bottr.me, youtube.com/@alkaid-art

laylavish commented:
@jenngra505 Hi! Thanks for the PR. I think I know what AI blocklist you are talking about. I assume you are talking about this one?:
```
https://codeberg.org/ranmaru22/ublacklist-anti-ai-art-subscription
```

Don't know why that one disappered, though. Weird. 

Anyways, everything looks okay, but I have a few issues. In multiple occassions in your uBlacklist commit you added an extra `*.` infront of the beginning match pattern `*://*.` This does not work, it errors out (syntax error); you can try it out yourself in the uBlacklist extension. I'm guessing that you wanted to get rid of all subdomains, sort of like a regular expression. This doesn't really matter since we would already match the second level domain in the first place by not adding a subdomain at all in the match pattern.

Also, you have created some unneeded duplicates. For example, `kakaobrain.com` is listed three times here (there was a misspelling with the other one `kakobrain.com` which is a site that doesn't exist as far as i can see).

Also, i'm wary of putting `123rf.com` in the main list purely because its a stock images site. I always put stock image sites in the nuclear list because they hold so many authentic images. Unfortunately however, these stock image sites are totally overrun by AI slop, but even so, I think it would be bad to put this in the main list because there's a high chance that people will believe that authentic images are AI generated even though they aren't, since many authentic images appear in Search. But for every authentic image there's AI slop right beside it, so I think the solution is to just use regex to catch many of the images that have text related to `AI generation` as possible, then add each outlier one by one, sort of like what I've done towards the bottom of the uBlacklist list with regex.

Also, please tell me if i'm wrong, but I believe that adding new entries for Adobe's `firefly.html` doesn't make much of a difference since we are already matching the base firefly directory.

Again thanks for the PR, you've done a great job! Just needs a little fine-tuning, is all.

jenngra505 commented:
I applied the necessary syntax fixes, removed resulting duplicates, and removed 123rf as it's already in the nuclear list. Hopefully, it's good to go.
